modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -230,26 +218,38 @@ a|Do not clear partition table on error.
 a|`-h`, `--help`
 a|Print help information.
 
+a| `-f`, `--image-file <path>`
+a|Specify a local image file manually. Used for debugging.
+
+a|
+a|
+
 2+|_Command-line argument_
 
 |*Argument* |*Description*
 
 a|`<device>`
 a|The destination device.
 
+a|
+a|
+
 2+|_coreos-installer embedded Ignition commands_
 
 |*Command* |*Description*
 
-a|`$ coreos-installer iso ignition embed <options> <ISO_image> <Ignition_config>`
+a|`$ coreos-installer iso ignition embed --ignition-file <options> <ISO_image>`
 a|Embed an Ignition config in an ISO image.
 
-a|`coreos-installer iso ignition show <options> <ISO_image> <Ignition_config>`
+a|`coreos-installer iso ignition show --ignition-file <options> <ISO_image>`
```

Comment:
This subcommand doesn't take `--ignition-file`.

Reply:
Revised: `coreos-installer iso ignition show <options> <ISO_image>`